albums/views.py

```diff
@@ -1,3 +1,122 @@
-from django.shortcuts import render
+import json
+import math
 
-# Create your views here.
+from django.http      import JsonResponse, HttpResponse
+from django.views     import View
+from django.db.models import Q
+
+from albums.models    import (Album, Genre, Artist, ReleaseType,
+                              PlayLink, AlbumPlayLink)
+
+class ListView(View): 
```

Comment:
어떤 resource의 list view 인지 명시해주는게 좋을 것 같습니다.

```suggestion
class AlbumㄴView(View): 
또는
class AlbumListView(View): 
```

Reply:
넵 수정하겠습니다 !